frontend/packages/dev-console/integration-tests/support/step-definitions/common/operators.ts

```diff
@@ -28,9 +22,18 @@ Given('user has installed Web Terminal operator', () => {
 });
 
 Given('user has installed OpenShift Serverless Operator', () => {
-  verifyAndInstallKnativeOperator();
+  verifyAndInstallOperator(operators.ServerlessOperator);
 });
 
 Given('user has installed OpenShift Pipelines Operator', () => {
-  verifyAndInstallPipelinesOperator();
+  verifyAndInstallOperator(operators.PipelinesOperator);
 });
+
+Given(
+  '{operator} operator is installed on the cluster in {string} namespace',
+  (operator: operators, namespace: string) => {
+    cy.logout();
+    cy.login(); // make sure we are logged in as kubeadmin
```

Comment:
Damn... I thought I removed them already...

Reply:
@makambalaji Hmm... It seems the login as kubeadmin is necessary here.
I just run the test locally without this part of code and it failed, because when second scenario was running, the logged-in user was still `test`, which is wrong - this user doesn't have enough rights to install operators etc.

Reply:
So far I don't know how to do this better. We could move the login from `before` hook to `beforeEach` hook which could solve this, but I think change like this should be done out of scope of this PR.

Reply:
before and beforehook changes affects other scripts, maybe we can keep as it is later if issue arises will check it